tests/psfSelectTest.py

```diff
@@ -189,21 +189,6 @@ def plantSources(x0, y0, nx, ny, sky, nObj, wid, detector, useRandom=False):
 
     return expos, goodAdded, expos0, goodAdded0
 
-#################################################################
-# quick and dirty detection (note: we already subtracted background)
-def detectAndMeasure(exposure, detConfig, measConfig):
```

Comment:
Please use a normal doc string instead of a comment, and preferably document the arguments (what classes are detConfig and measConfig instances of?)